oeis.py

```diff
@@ -139,6 +138,22 @@ def A000045(start: int = 0, limit: int = 20) -> Collection[int]:
         sequence.append(sequence[i - 1] + sequence[i - 2])
     return sequence
 
+@oeis
+def A000566(start: int = 0, limit: int = 20) -> Collection[int]:
+    def aList():
+
+        x, y = 1, 1
+
+        yield 0
+
+        while True:
+
+            yield x
+
+            x, y = x + y + 5, y + 5
```

Comment:
Please avoid writing on every other line, compact this, and run black just to add mandatory blank lines.